src/components/AvatarWithIndicator.js

```diff
@@ -1,126 +1,66 @@
-    isActive: false,
     size: 'default',
-    isSyncing: false,
     tooltipText: '',
+    policiesMemberList: {},
 };
 
-class AvatarWithIndicator extends PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.animation = new SpinningIndicatorAnimation();
-    }
-
-    componentDidMount() {
-        if (!this.props.isSyncing) {
-            return;
-        }
-
-        this.animation.start();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.isSyncing && this.props.isSyncing) {
-            this.animation.start();
-        } else if (prevProps.isSyncing && !this.props.isSyncing) {
-            this.animation.stop();
-        }
-    }
-
-    componentWillUnmount() {
-        this.animation.stop();
-    }
-
-    /**
-     * Returns user status as text
-     *
-     * @returns {String}
-     */
-    userStatus() {
-        if (this.props.isSyncing) {
-            return this.props.translate('profilePage.syncing');
-        }
-
-        if (this.props.isActive) {
-            return this.props.translate('profilePage.online');
-        }
-
-        if (!this.props.isActive) {
-            return this.props.translate('profilePage.offline');
-        }
-    }
-
-    render() {
-        const indicatorStyles = [
-            styles.alignItemsCenter,
-            styles.justifyContentCenter,
-            this.props.size === 'large' ? styles.statusIndicatorLarge : styles.statusIndicator,
-            this.props.isActive ? styles.statusIndicatorOnline : styles.statusIndicatorOffline,
-            this.animation.getSyncingStyles(),
-        ];
-
-        return (
-            <View
-                style={[this.props.size === 'large' ? styles.avatarLarge : styles.sidebarAvatar]}
-            >
-                <Tooltip text={this.props.tooltipText}>
-                    <Avatar
-                        imageStyles={[this.props.size === 'large' ? styles.avatarLarge : null]}
-                        source={this.props.source}
-                        size={this.props.size}
-                    />
-                </Tooltip>
-                <Tooltip text={this.userStatus()} absolute>
-                    <Animated.View style={StyleSheet.flatten(indicatorStyles)}>
-                        {this.props.isSyncing && (
-                            <Icon
-                                src={Expensicons.Sync}
-                                fill={themeColors.textReversed}
-                                width={6}
-                                height={6}
-                            />
-                        )}
-                    </Animated.View>
-                </Tooltip>
-            </View>
-        );
-    }
-}
+const AvatarWithIndicator = (props) => {
+    const indicatorStyles = [
+        styles.alignItemsCenter,
+        styles.justifyContentCenter,
+        props.size === 'large' ? styles.statusIndicatorLarge : styles.statusIndicator,
+    ];
+    const isLarge = props.size === 'large';
```

Comment:
move `const isLarge = props.size === 'large';` before `const indicatorStyles ` and use it in `props.size === 'large' ? styles.statusIndicatorLarge : styles.statusIndicator,`